xml/System.IO.Compression/CompressionLevel.xml

```diff
@@ -83,7 +83,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>The compression operation should complete as quickly as possible, even if the resulting file is not optimally compressed.</summary>
```

Comment:
In the PR description you write:

> whereas Fastest is best speed regardless of compression ratio

and here you update `Fastest` description (`MemberValue=1`) to be the same as `Optimal` (`MemberValue=0`):

> optimally balance compression speed and output size.

I don't understand?

Reply:
This line shouldn't have been edited.